### ISSUE TITLE
Consider creating a `startup-modals` component 

### ISSUE THREAD

**Issue by dianabarsan** [medic/angular10-migration#38]
To replace the direct interaction between the `privacy-policy` component and `startup-modals` service. 
https://github.com/medic/cht-core/blob/4750-angular-10-take-2/webapp/src/ts/modules/privacy-policy/privacy-policy.component.ts#L54

**Comment by dianabarsan** [comment]
On second though, maybe it's not worth the extra code + rendering time to have this additional component and the direct interaction makes things more obvious.

**Comment by mrsarm** [comment]
Yes, to sync all in that way I think we should move all the state to Redux, so the app component or the services called from the app component does the checks changing the state and then the components subscribed to  those state variables do the rendering. Currently some of the modals use Redux some others not, and not sure if worth the extra rendering work as you said.

**Comment by latin-panda** [comment]
Removing ticket from current iteration, we can reevaluate this one later as per chat in slack :)
